modules/apigee_edge_teams/src/Plugin/EntityReferenceSelection/TeamMembersSelection.php

```diff
@@ -222,4 +229,21 @@ protected function buildEntityQuery($match = NULL, $match_operator = 'CONTAINS')
     return $query;
   }
 
+  /**
+   * {@inheritdoc}
+   */
+  public function getReferenceableEntities($match = NULL, $match_operator = 'CONTAINS', $limit = 0) {
+    $options = parent::getReferenceableEntities($match, $match_operator, $limit);
```

Comment:
Do not call parent because if you do then you do all the stuff twice (including loading entities, which should be cached but if it does not it extremely slows down the autocomplete).

Reply:
The parent implementation is not statically caching the results, so I moved the implementation into this class so it's not calling `parent` anymore.